### PR TITLE
Auto spell check documentation

### PR DESCRIPTION
**Pull request by InnocentBug**
Also enable automatic spell checking of documentation.

### CODE REVIEW

**Comment by InnocentBug** [comment]
This isn't perfect, but at least it verifies that individual words are spelled correctly.

File: pysages/methods/metad.py [hunk 1]
```diff
@@ -20,34 +20,34 @@
 
 class MetadynamicsState(NamedTuple):
     """
-    Metadynamics helper state
+    Meta-dynamics helper state
```
[the remark this answers is not in the document]
Comment (InnocentBug) [reply pysages/methods/metad.py]: yeah, ok with me

File: pysages/methods/metad.py [hunk 2]
```diff
@@ -104,22 +104,22 @@ class Metadynamics(SamplingMethod):
...
 
     grid: Optional[Grid] = None
         If provided, it will be used to accelerate the computation by
         approximating the bias potential and its gradient over its centers.
 
     kB: Optional[float]
-        Boltzmann constant. Must be provided for well-tempered metadynamics
-        simulations and should match the internal units of the backend.
+        Boltzmann constant. Must be provided for well-tempered Meta-dynamics
+        simulations and should match the internal units of the back end.
```
[the remark this answers is not in the document]
Comment (InnocentBug) [reply pysages/methods/metad.py]: sure

**Review by pabloferz — APPROVED** [review]

**Comment by InnocentBug** [comment]
Unortunately, it doesn't spell check our readmes :(

**Pull request merged**